highway_env/envs/intersection_env.py

```diff
@@ -235,6 +235,7 @@ def default_config(cls):
                 "stack_size": 4,
                 "observation_shape": (84, 84)
             } ,
+            "n_vehicles": 10,
```

Comment:
Could you please rename it as vehicles_count for consistency with other environments ?